Upstream changed the default rootfs to tmpfs when none has been passed
to the kernel - this doesn't fit our purposes, so change it back.

Signed-off-by: Imre Kaloz <kaloz@openwrt.org>

--- a/init/do_mounts.c
+++ b/init/do_mounts.c
@@ -625,6 +625,7 @@ int __init init_rootfs(void)
 	if (err)
 		return err;
 
+#if 0
 	if (IS_ENABLED(CONFIG_TMPFS) && !saved_root_name[0] &&
 		(!root_fs_names || strstr(root_fs_names, "tmpfs"))) {
 		err = shmem_init();
@@ -632,6 +633,9 @@ int __init init_rootfs(void)
 	} else {
 		err = init_ramfs_fs();
 	}
+#else
+	err = init_ramfs_fs();
+#endif
 
 	if (err)
 		unregister_filesystem(&rootfs_fs_type);
